pages.zh/common/nmap.md

```diff
@@ -0,0 +1,37 @@
+# nmap
+
+> 网络探索工具和安全/端口扫描程序。
+> 仅当以特权运行Nmap时，某些功能才激活。
+> 更多信息见: <https://nmap.org>.
```

Comment:
I think this is the correct period.
```suggestion
> 更多信息见: <https://nmap.org>。
```

Reply:
You're right!

Reply:
Actually, per #5240 we've decided to use the English period at the end of the more information link sentence, because the link and it's surrounding `<` and `>` are all halfwidth, so it makes sense to use a halfwidth period (`.`) there.

Reply:
@bl-ue OK.